### ISSUE TITLE
⌨ DEI, Profile, no Indigenous community text has insufficient colour contrast

### ISSUE THREAD

**Issue by esizer** [GCTC-NTGC/gc-digital-talent#6760]
## ⌨ Accessibility Issue

The text on the user profile "You haven't identified your community yet!..." is using `primary` colour which does not have sufficient contrast when compared against its background colour.

### WCAG 2.1 Criteria

[Elements must have sufficient color contrast](https://dequeuniversity.com/rules/axe/4.7/color-contrast?application=axeAPI)

## 🦋 Expected Behaviour

The text should be using a slightly darker shade of the `primary` colour.

## 📋 Steps to Reproduce

1. Run storybook (it has the proper state)
2. Open the "User profile story 5" story
3. Confirm the text in DEI is not at the minimum contrast requirement (4.5:1)

## 📸 Screenshot

![Screenshot 2023-05-31 090626](https://github.com/GCTC-NTGC/gc-digital-talent/assets/4127998/2873fc97-c892-447e-8d53-82800a5cad96)

## 🙋‍♀️ Proposed Solution

Update the colour to use `primary.dark`

## ✅ Acceptance Criteria

A set of assumptions which, when tested, verify that the issue was addressed.

- [ ] Text is updated to use a darker shade of `primary`
- [ ] Text passes the min, contrast ratio (4.5:1)
